public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -650,6 +650,9 @@ export const Explorer = ({
                         requestParams={requestParams}
                         startTime={startTime}
                         endTime={endTime}
+                        isNotDefaultDatasource={
```

Comment:
Slightly odd to name it `isNotDefaultDataSource` when most uses negate it. Might be clearer to use `isDefaultDataSource` and flip all the conditions.

Reply:
Is it even better to name it `is(not)?Default` instead of `is(not)?S3`?

Reply:
True, I made this prop with all of the changes in mind but when I ended up fixing everything, I ended up needing to negate it. Changing it to isDefaultDataSource. I'm not sure if I want to name it with s3 explicitly because my aim is for the logic to work whenever the data grid is used on anything that is not opensearch indices.